File: Templates/java/Class Template.java
public class MyClass {
    private String name;

    public MyClass(String name) {
        this.name = name;
    }

    public void greet() {
        System.out.println("Hello, " + name + "!");
    }

    public static void main(String[] args) {
        MyClass obj = new MyClass("User");
        obj.greet();
    }
}

File: Templates/cpp/ClassTemplate.cpp
#include <iostream>
#include <string>

class MyClass {
public:
    MyClass(const std::string& name) : name(name) {}
    void greet() const {
        std::cout << "Hello, " << name << "!" << std::endl;
    }

private:
    std::string name;
};

int main() {
    MyClass obj("User");
    obj.greet();
    return 0;
}

